packages/quantic/package.json

```diff
@@ -13,9 +13,9 @@
   "scripts": {
     "test": "npm run build:staticresources && npm run lint:check:tests && npm run validate:types && npm run test:unit",
     "lint:check:tests": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
-    "lint:fix": "eslint force-app/main/default/lwc/ --fix && eslint force-app/examples/main/lwc/ --fix",
-    "lint:prettier:fix": "prettier \"force-app/{,**}/*.{cls,trigger,js}\" --write",
-    "fix": "npm run lint:fix && npm run lint:prettier:fix",
+    "lint:fix": "npm run lint:fix:js",
+    "lint:fix:js": "eslint force-app/main/default/lwc/ --fix && eslint force-app/examples/main/lwc/ --fix && prettier \"force-app/{,**}/*.js\" --write",
```

Comment:
nit: having the cmd flags e.g. `--fix` before the path arg would make it a little bit more readable